content/3_Silver/Two_Pointers.mdx

```diff
@@ -44,15 +45,145 @@ of an interval. It can also be used to track two values in an array as shown in
 
 We want to find the longest contiguous segment of books that can be read within $t$ minutes. 
 
-To accomplish this, we can define $\texttt{left}$ and $\texttt{right}$ as the start and end of the segment respectively and use two pointers to track the interval.
+To accomplish this, we can define $\texttt{left}$ and $\texttt{right}$ to **represent** the beginning and end of the **segment**. However, both will start at the beginning of the array. These numbers can be thought of as pointers, hence the name "two pointers".
 
 For every $\texttt{left}$ index, we need to increase our $\texttt{right}$ pointer until the total time for that segment is the maximum while being less than $t$.
 
 We will keep track of the max segment size with $\texttt{ans}$, and take the maximum $\texttt{right} - \texttt{left}$ value.
```

Comment:
Shoudn't this be the maximum value of right - left + 1?

Reply:
Yeah, it should.
```suggestion
We will keep track of the max segment size with $\texttt{ans}$, and take the maximum $\texttt{right} - \texttt{left}+1$ value.
```